### PR TITLE
deb: include man pages

### PR DESCRIPTION
**Pull request by thaJeztah**
closes https://github.com/docker/containerd-packaging/pull/8
fixes https://github.com/docker/containerd-packaging/issues/4
fixes https://github.com/docker/containerd-packaging/issues/69

### CODE REVIEW

File: rpm/containerd.spec [hunk 1]
```diff
@@ -121,11 +121,13 @@ install -D -m 0755 bin/* %{buildroot}%{_bindir}
 install -D -m 0644 %{S:1} %{buildroot}%{_unitdir}/containerd.service
 install -D -m 0644 %{S:2} %{buildroot}%{_sysconfdir}/containerd/config.toml
 
-# install manpages
-install -d %{buildroot}%{_mandir}/man1
-install -p -m 644 man/*.1 $RPM_BUILD_ROOT/%{_mandir}/man1
-install -d %{buildroot}%{_mandir}/man5
-install -p -m 644 man/*.5 $RPM_BUILD_ROOT/%{_mandir}/man5
+# install manpages, taking into account that not all sections may be present
+for i in $(seq 1 8); do
```
Comment (chris-crone) [inline rpm/containerd.spec:125]: I see that we used to fill man 1 and 5 and that we're now doing 1 and 8, was this change intended?

Comment (thaJeztah) [reply rpm/containerd.spec:125]: Yes; the man-pages were in the wrong section (1 instead of 8), this was fixed upstream, but now the packaging script here didn't take section 8 into account.

I changed it to take all sections (1..8) into account, so that we won't have to update this again if new man pages are added in future that are in different sections
